topic_folders/governance/lesson-program-policy.md

```diff
@@ -43,7 +43,7 @@ This Policy outlines the Roles and Responsibilities of the Lesson Programs and T
 - The Carpentries will include the Lesson Programs in assessment efforts, including survey development and data analysis. 
 - The Carpentries will ensure the instructor training program addresses the training needs of the Lesson Program, incorporating examples and exercises targeted towards the communities served by the Lesson Program.
 - The Carpentries will prepare Trainers to work with trainees from the Lesson Program. 
-- The Carpentries Instructor Training recruitment and retention strategies will include building communities of InstructorsIinstructors in communities and domains served by the Lesson Program. 
+- The Carpentries Instructor Training recruitment and retention strategies will include building communities of Instructors in domains served by the Lesson Program. 
```

Comment:
Please esp. review this change. My suggestion is not the only valid resolution here.